asserts/gpgkeypairmgr_test.go

```diff
@@ -338,3 +338,19 @@ func (gkms *gpgKeypairMgrSuite) TestList(c *C) {
 	c.Check(keys[0].ID, Equals, assertstest.DevKeyID)
 	c.Check(keys[0].Name, Not(Equals), "")
 }
+
+func (gkms *gpgKeypairMgrSuite) TestDelete(c *C) {
+	c.Skip("produces a UI prompt")
```

Comment:
I think we want the prompt for real usage, but maybe I can find a way to inject the yes only for the tests. It's a bit strange but so far we didn't have many gpg tests because of that. Anyway I need to see if --yes helps here, it seems not to work for everything.

Reply:
done something hopefully not too unreasonable